src/Components/test/testassets/Components.TestServer/RazorComponents/GlobalInteractivityApp.razor

```diff
@@ -19,16 +16,15 @@
         });
     </script>
 </body>
-
 </html>
 
-
 @code {
     [CascadingParameter]
     private HttpContext HttpContext { get; set; } = default!;
 
-    // Statically render pages in the "/Account" subdirectory like we do in the Blazor Web template with Individaul auth.
-    private IComponentRenderMode? RenderModeForPage => HttpContext.Request.Path.StartsWithSegments("/static")
+    // Statically render a specific page to show we can override the rendermode based on URL
+    // Elsewhere, use global interactivity.
+    private IComponentRenderMode? RenderModeForPage => HttpContext.Request.Path.StartsWithSegments("/globally-interactive/static-via-url")
         ? null
-        : RenderMode.InteractiveAuto;
+        : RenderMode.InteractiveWebAssembly;
```

Comment:
Changing this from Auto to WebAssembly to make the E2E tests more deterministic, and because WebAssembly is the more complicated case anyway.